subscription/src/main/java/org/killbill/billing/subscription/api/SubscriptionBaseApiService.java

```diff
@@ -107,4 +107,7 @@ public List<SubscriptionBaseEvent> getEventsOnCancelPlan(final DefaultSubscripti
                                                              final InternalTenantContext internalTenantContext) throws CatalogApiException;
 
     boolean undoChangePlan(DefaultSubscriptionBase defaultSubscriptionBase, CallContext context) throws SubscriptionBaseApiException;
+
+    int getBCD(InternalTenantContext context) throws SubscriptionBaseApiException;
```

Comment:
Rename it to `getAccountBCD`?

Reply:
Fixed in https://github.com/killbill/killbill/pull/1663/commits/d5fb66fb4deb852b0f4e712d8829f0824e81d57a